sigma/backends/loki/loki.py

```diff
@@ -279,6 +283,12 @@ def set_expression_templates(negated: bool) -> None:
         expression was negated or not."""
         if negated == LogQLBackend.current_templates:
             return  # nothing to do!
+
+        # Set the expression templates regardless of the negation state
+        LogQLBackend.compare_op_expression = "{field}{operator}{value}"
+        LogQLBackend.field_exists_expression = '{field}!=""'

```

Comment:
These need to be flipped if they are negated (as confusing as it would be if someone wrote a rule like that! :laughing: ). I.e.:

```
sel:
  fieldA|exists: yes
condition: not sel
```

Should result in the query containing:
```
| fieldA=""
```

Reply:
Addressed in https://github.com/grafana/pySigma-backend-loki/pull/151/commits/1257ae74cf1d66f1b07c0a0f44e832d19c03c5a9 and https://github.com/grafana/pySigma-backend-loki/pull/151/commits/427c744bd9a59e9bfee872f7fcdb0cabe255a418.